src/test/resources/config/spark/w_all_tpcds-dependent-delta.yaml

```diff
@@ -0,0 +1,34 @@
+# Description: Workload for test: All task types, TPC-DS, Delta
```

Comment:
We should also execute this as part of the CI on the Spark container. Please add a new test in `DriverSparkTest` (similar to `testJDBCTPCDSAllTasksDelta`).

Reply:
Is this still valid if Spark throws an unsupported exception now?

Reply:
Yes, the test I was suggesting would go through JDBC (this is controlled by the annotation on the method as you will be able to see), which is the implementation we support.
Optionally, we can also add a similar test that will expect an unsupported exception for session client implementation.

Reply:
I changed the test setup to add a dependent data maintenance phase to all three TPC-DS tests (delta/iceberg/hudi) that were already present which I think is the right way to resolve it. Let me know what you think.